Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/calculation/JsonFunctionsWrapper.kt

```diff
@@ -0,0 +1,56 @@
+package de.rki.coronawarnapp.ccl.dccwalletinfo.calculation
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import de.rki.coronawarnapp.ccl.configuration.model.CCLConfiguration
+import de.rki.coronawarnapp.ccl.configuration.storage.CCLConfigurationRepository
+import de.rki.coronawarnapp.util.serialization.BaseJackson
+import de.rki.jfn.JsonFunctions
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class JsonFunctionsWrapper @Inject constructor(
```

Comment:
Maybe `JsonFunctionsEngine` or `JsonFunctionsEvalutaor` is a better name?

Reply:
I renamed it to CclJsonFunctions, as it is specific now